docs/ides/web-ides.md

```diff
@@ -22,6 +22,12 @@ resource "coder_app" "portainer" {
   icon          = "https://simpleicons.org/icons/portainer.svg"
   url           = "http://localhost:8000"
   relative_path = true
+
+  healthcheck {
+    url       = "http://localhost:8000/"
```

Comment:
does portainer have a healthcheck route? If they have a helm chart you can see what they use for kubernetes.

Reply:
Ah good call.